tag-security-and-privacy.md

```diff
@@ -0,0 +1,75 @@
+# TAG Review: Security and Privacy questionnaire
+
+## 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+There is an ongoing discussion ([tc39/proposal-import-attributes#61](https://github.com/tc39/proposal-import-attributes/issues/61)) about communicating the `type` attribute to the origin. For instance:
+
+```js
+import json from "/foo.json" with type: "json";
+```
+
+would presumably send `Accept: application/json` to the origin.
```

Comment:
if it does, this allows the server to easily provide a different module based on the type, which violates the intended restriction that was agreed to for stage 2.

Reply:
The HTML integration is under discussion in https://github.com/whatwg/html/issues/5640 . There is no header sent in the current proposal, and no one on the thread is advocating to add a header. Let's hash out these things in relevant issues.

Reply:
Yeah to be clear, i'm not advocating for it, I wanted to be complete for the security/privacy review since it's being discussed

Reply:
Suggested wording: This feature only exposes the information to web servers that a particular resource is being fetched. This is necessary to load the module graph. The current proposal does not send any headers related to the import condition/module type for content negotiation, though this has been raised in #61.